iotdb-core/metrics/core/src/main/resources/META-INF/services/org.apache.iotdb.metrics.AbstractMetricManager

```diff
@@ -15,4 +15,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-org.apache.iotdb.metrics.micrometer.MicrometerMetricManager
```

Comment:
Maybe there is no need to use SPI if we choose micrometer as implementation

Reply:
good idea!